libs/partners/mistralai/langchain_mistralai/embeddings.py

```diff
@@ -209,16 +211,27 @@ def embed_documents(self, texts: List[str]) -> List[List[float]]:
             List of embeddings, one for each text.
         """
         try:
-            batch_responses = (
-                self.client.post(
+            batch_responses = []
+
+            @retry(
+                retry=retry_if_exception_type(Exception),
```

Comment:
Generally, retries should never be implemented on 4xx errors (except for 408 and 429). e.g., 403 should not be retried by default

* What do we do in other parts of the code? Perhaps there's a better example that can be adopted?
* What do other models do in the code base in terms of exposing the retry parameters so users can adjust? (e.g.,what if someone wants to have the first retry after 1 second rather than 30 seconds?)

Reply:
- @eyurtsev it is a 429 error therefore retrying makes sense.
- I was not able to find any related example.
- Yes I can make the wait and stop seconds as parameters.